Sources/Sentry/include/SentryTraceContext.h

```diff
@@ -1,9 +1,18 @@
-#import "SentryId.h"
-#import "SentrySerializable.h"
+#if __has_include(<Sentry/SentrySerializable.h>)
+#    import <Sentry/SentrySerializable.h>
+#else
+#    import "SentrySerializable.h"
+#endif
+
+#if __has_include(<Sentry/SentryId.h>)
+#    import <Sentry/SentryId.h>
+#else
+#    import "SentryId.h"
+#endif
 
 NS_ASSUME_NONNULL_BEGIN
 
-@class SentryScope, SentryOptions, SentryTracer, SentryUser, SentryBaggage;
+@class SentryScope, SentryId, SentryOptions, SentryTracer, SentryUser, SentryBaggage;
```

Comment:
We wouldn't need this forward class dec since SentryId.h is always being imported one way or another